src/main/java/com/deusgmbh/xcusatio/context/wildcard/CalendarContext.java

```diff
@@ -0,0 +1,56 @@
+package com.deusgmbh.xcusatio.context.wildcard;
+
+import com.deusgmbh.xcusatio.data.lecturer.Lecturer;
+
+public class CalendarContext {
+    private LectureEvent lectureEvent;
+    private String minutesLeft;
+    private String minutesPassed;
+
+    class LectureEvent {

```

Comment:
Diese Klassen kannst du auch ruhig in eine eigene Datei verschieben und public machen. Außer die gehen die "Außenwelt" wirklich nichts an.